### PR TITLE
Update PUNDIX token

### PR DESCRIPTION
**Pull request by zakir-code**
### Adding Assets

If adding a new asset, please ensure the following:
- [ ] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [x] Add asset to bottom of `zone_assets.json`.
   - [x] `chain_name` and `base_denom` are provided and use values exactly as defined at the Chain Registry.
   - [x] `path` is provided, and the IBC channel referenced is registered at the Chain Registry (unless native to Osmosis).
   - [x] `osmosis_verified` is set to `false`
   - [x] `listing_date_time_utc` is specified and accurate
   - [ ] Optional: `transfer_methods`, `peg_mechanism`, `override_properties`, `canonical`, `categories`, where necessary (see [README](https://github.com/osmosis-labs/assetlists/tree/main?tab=readme-ov-file#how-to-add-assets) for details).

### Upgrading Asset to Verified

If upgrading an Asset to Verified, please see the requirements specified at [LISTING](https://github.com/osmosis-labs/assetlists/blob/main/LISTING.md#upgrade-asset-to-verified-status-permissioned), and ensure the following:
- [x] Asset is defined thoroughly at the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - [x] A meaningful `description` and `extended_description`.
   - [x] Associated `socials`, including `website` and `twitter`.
   - [x] Logo Image has a square Aspect Ratio and < 250 KB file size.
- [ ] This pool contains contains at least $1k USD-worth of liquidity of the asset (Provide Pool ID): ______

'Verified' Status Validation Checklist (to be completed by Osmosis Zone maintainers):
- [ ] Verify appearance and metadata
- [ ] Accurate Price
- [ ] Trading and routing functionality
- [ ] Withdraw and Deposit
   - [ ] Deposit Transaction URL (if in-app)

### CODE REVIEW

**Comment by zakir-code** [comment]
Ref: https://github.com/osmosis-labs/assetlists/pull/1440

**Comment by sunnya97** [comment]
Is there a pool for PUNDIX?

**Comment by JeremyParish69** [comment]
Are we able to get some PUNDIX liquidity over to Osmosis? If so, I think we can proceed with making both PUNDIX and PURSE verified--otherwise, I'll close these PRs for now.

**Comment by zakir-code** [comment]
> Are we able to get some PUNDIX liquidity over to Osmosis? If so, I think we can proceed with making both PUNDIX and PURSE verified--otherwise, I'll close these PRs for now.

Can we transfer pundix token from fxcore ibc to osmosis and add liquidity.

**Comment by zakir-code** [comment]
> > > Are we able to get some PUNDIX liquidity over to Osmosis? If so, I think we can proceed with making both PUNDIX and PURSE verified--otherwise, I'll close these PRs for now.
> > 
> > 
> > Can we transfer pundix token from fxcore ibc to osmosis and add liquidity.
> 
> Are you asking if it's possible? I would assume so because PUINDIX is registered and listed, but I'm requesting that you have someone bring some $PUNDIX liquidity to Osmosis so we can proceed.

Yes, we want to replace the pundix token on the osmosis chain. The updated pundix token is cross-chain(IBC) from the fxcore chain to the osmosis chain.

https://github.com/cosmos/chain-registry/pull/4667

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -3439,17 +3439,24 @@
...
+          "name": "Starscan",
+          "type": "external_interface",
+          "deposit_url": "https://starscan.io/fxbridge?from=fxcore&to=osmosis&token=PUNDIX"
+        }
+      ],
       "canonical": {
         "chain_name": "ethereum",
         "base_denom": "0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38"
       },
+      "osmosis_verified": true,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:3456]: Can I ask that we start with updating this PR to keep verified as 'false' so we can validate these changes first?
Then next PR we can set to 'true'.
The osmosis_unlisted property can be removed in either PR, just replace it with a listed date when we do so.

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -3439,17 +3439,24 @@
       "_comment": "BackBone Labs Liquid Staked OSMO $bOSMO"
     },
     {
-      "chain_name": "pundix",
-      "base_denom": "ibc/55367B7B6572631B78A93C66EF9FDFCE87CDE372CC4ED7848DA78C1EB1DCDD78",
-      "path": "transfer/channel-12618/transfer/channel-0/eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38",
+      "chain_name": "fxcore",
+      "base_denom": "ibc/46D8D1A6E2A80ECCB7CA6663086A2E749C508B68DA56A077CD26E6F4F9691EEE",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:3443]: This is incorrect. It's be base denom on Osmosis, but that's not what's needed here. The base denom on fxcore is 'eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38'

File: osmosis-1/osmosis.zone_assets.json (hunk 1 above)
Comment (zakir-code) [reply osmosis-1/osmosis.zone_assets.json:3456]: ok

**Comment by JeremyParish69** [comment]
I'm finding that when f(x)core and Osmosis are selected on starscan, the asset PUNDIX is no longer selectable.
When ready, please create a new PR to set to verified, but note that I cannot test this without some liquidity.

**Comment by zakir-code** [comment]
okey

**Pull request merged**